src/AutoPilotPlugins/Common/MotorComponent.cc

```diff
@@ -48,7 +48,12 @@ QStringList MotorComponent::setupCompleteChangedTriggerList(void) const
 
 QUrl MotorComponent::setupSource(void) const
 {
-    return QUrl::fromUserInput(QStringLiteral("qrc:/qml/MotorComponent.qml"));
```

Comment:
This should be left alone and the ArduPilot code should have it's own implementation.